roles/pulp/templates/settings.py.j2

```diff
@@ -1,7 +1,21 @@
 CONTENT_ORIGIN="http://{{ ansible_hostname }}:8080"
+API_CONTENT_ORIGIN="http://{{ ansible_hostname }}:24817"
+CONTENT_SERVICE_ORIGIN="http://{{ ansible_hostname }}:24816"
 CACHE_ENABLED=True
 REDIS_HOST="localhost"
 REDIS_PORT=6379
```

Comment:
Now that it's no longer using the included Redis, but the system one I think it may use the default DB. Foreman may do the same. In puppet-pulpcore we explicitly chose a DB number that we don't use elsewhere and I think we should mirror that.

Reply:
https://github.com/theforeman/puppet-pulpcore/blob/26254db23ee9456520049a4651845fecc6e2cbe9/manifests/init.pp#L241 says we use `8`
so:
```suggestion
REDIS_HOST="localhost"
REDIS_PORT=6379
REDIS_DB=8
```